CenterForOpenScience/exp-addons
# Style checking and code cleanup

Refs: https://openscience.atlassian.net/browse/LEI-315
Companion to: 

## Purpose
Before we reorganize our addons, we should bring them up to speed with current development standards and remove any unused code.

## Summary of changes
- Activate JSCS style checker
- Activate template linting
- Migrate all components to pod styles for easier rearranging later

## Testing notes
In theory everything should work exactly the same afterwards.

Verify no missing images and no console errors from unrunnable code. Check for display artifacts due to extra spacing etc.

- Run through sample lookit experiment. 
- Run through sample ISP experiment
